src/main/java/core/basesyntax/ToDraw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface ToDraw {
```

Comment:
Interface names should be adjectives (most preferable) or nouns (as alternative).
How about `Drawable`?